internal/files/linkedfiles_test.go

```diff
@@ -845,13 +855,13 @@ func TestLinksFS_WorkDirValidation(t *testing.T) {
 			name:        "invalid absolute workDir outside repo",
 			workDir:     outsideDir,
 			expectError: true,
-			errorMsg:    "is outside the repository root",
+			errorMsg:    "path escapes from parent",
```

Comment:
Error from the stdlib is directly returned now, but we cannot check for it to convert it as it is not public (https://github.com/golang/go/blob/e5502e0959bb54ec70ca500e8d2b6f5ac5efbc53/src/os/file.go#L421).

We could check the error message and replace it, but not sure if it is worth it.